backend/tests/review/test_stats.py

```diff
@@ -312,6 +312,14 @@ def test_course(self):
             "course-plots", "ESE-120", self.course_plots_subdict,
         )
 
+        instructor_ids = ",".join([str(id) for id in [Instructor.objects.get().pk]])
```

Comment:
Might also want to test for the case where only non-existent ID(s) are passed, to simultaneously test narrowing the data returned and the "no matches" edge case, using something like `empty_course_plots_subdict` from below